x11-misc/xflux/xflux-20130927.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Makes the color of your computer's display adapt to the time of the day"
+HOMEPAGE="https://justgetflux.com/"
+SRC_URI="amd64?  ( https://justgetflux.com/linux/${PN}64.tgz -> ${PN}64-${PV}.tar.gz )
+	x86?	( https://justgetflux.com/linux/${PN}-pre.tgz -> ${P}.tar.gz )"
```

Comment:
it seems to me that you have more than one space here

Reply:
Thanks! Is now fixed.